docs/cli.md

```diff
@@ -26,19 +26,24 @@ needs to be required in your support files and globally installed modules cannot
   * To escape special regex characters in scenario name, use backslash e.g., `\(Scenario Name\)`
 * Use [Tags](#tags)
 
-## Requiring support files
+## Loading support files
 
-By default, the following files are required:
+By default, the following files are loaded:
 * If the features live in a `features` directory (at any level)
-  * `features/**/*.js`
+  * `features/**/*.(js|cjs|mjs)`
 * Otherwise
-  * `<DIR>/**/*.js` for each directory containing the selected features
+  * `<DIR>/**/*.(js|cjs|mjs)` for each directory containing the selected features
 
-Alternatively, you can use `--require <GLOB|DIR|FILE>` to explicitly require support files before executing the features. Uses [glob](https://github.com/isaacs/node-glob) patterns.
+With the defaults described above, `.js` files are loaded via `require()`, whereas `.cjs` and `.mjs` files are loaded via `import()`.
```

Comment:
`.cjs` files shouldn't be loaded using `require()` too?

Reply:
CommonJS files can be `import()`ed fine. The reason we're avoiding it with `.js` is because the user might be relying on some transpilation and we don't support that yet with `import()`. To be honest I think this just adds confusion so I'll remove `.cjs` from the default handling, users can always specify it for either `require` or `import`.

Reply:
Ok, let's do that